cli/kubectl-hyperkube

```diff
@@ -330,18 +347,33 @@ def get_cluster_metadata(ctx, cluster):
     """Get cluster metadata for a give cluster name"""
 
     try:
-        r = requests.get(
-            (f'{ctx.obj["url"]}/{ctx.obj["stage"]}'
-             f'/clusters/get-cluster-metadata?cluster_name={cluster}'),
-            headers=ctx.obj['headers']
-        )
+        r = _build_request(ctx, "GET", "/clusters/get-cluster-metadata",
+                           query_string=f'?cluster_name={cluster}')
         hkc_print(r.json())
         if r.status_code == 404:
             sys.exit(1)
     except requests.exceptions.RequestException as err:
         print(f'Request error: {err}')
 
 
+def _build_request(ctx, verb, api_endpoint,
```

Comment:
This method will make things way cleaner! Thanks. However, it does more than build the request. Why not just call it `_request`?

Reply:
I like that! yeah, I was thinking about that too. Ok, will switch up name